vNext/AISKU/README.md

```diff
@@ -181,33 +185,36 @@ If you are using the current application insights PRODUCTION SDK (1.0.20) and wa
 	Call downloadAndSetup to download full ApplicationInsights script from CDN and initialize it with instrumentation key.
 ```ts
 appInsights.downloadAndSetup({
-	instrumentationKey: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx",
-	url: "https://az416426.vo.msecnd.net/beta/ai.2.min.js"
+    instrumentationKey: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx",
+    url: "https://az416426.vo.msecnd.net/beta/ai.2.min.js"
 });
 ```
 Test in internal environment to verify monitoring telemetry is working as expected. If all works, please update your api signatures appropriately to SDK V2 version and deploy in your production environments.
 
-## Build a new extension for the SDK:
+## Build a new extension for the SDK
 The beta SDK supports the ability to include multiple extensions at runtime. In order to create a new extension, please implement the following interface:
 
-	> ITelemetryPlugin
+	- [ITelemetryPlugin](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/vNext/shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/ITelemetryPlugin.ts)
```

Comment:
Looks like this isn't rendering properly. Probably need to remove the indentation